cadc-util/src/main/java/ca/nrc/cadc/net/DigestUtil.java

```diff
@@ -200,11 +201,18 @@ public static String base64Encode(String value) {
      * @param value Value to decode.
      * @return Base64 decode of the value.
      */
-    public static String base64Decode(String value) {
+    public static String base64Decode(String scheme, String value) {
```

Comment:
Change in API - hopefully no other packages are using this directly. I did a quick check locally and that seems to be the case.

Reply:
yes, in my full change to correctness I actually made these methods private. API methods that take `String` and have vague javadocs are really dangerous and should be kept private where possible.

anyway, private didn't seem to break anything.